app/assets/src/components/views/SampleViewV2/report_table.scss

```diff
@@ -57,28 +57,38 @@
   font-weight: $font-weight-semibold;
   text-align: left;
 
-  .taxonName {
-    text-transform: capitalize;
-  }
-
   .expandIcon {
     flex: 0 0 20px;
   }
 
-  .missingName {
-    color: $medium-grey;
-  }
-}
+  .taxonContainer {
+    .taxonName {
+      text-transform: capitalize;
```

Comment:
Add `word-wrap` to the taxonName as well? Currently only the `countInfo` text is being wrapped.

Reply:
I will add it for now. But wrapping the name of the taxon might be undesired because it will push the - soon to be added - action icons outside of the cell.